hack/test-cmd.sh

```diff
@@ -257,6 +251,12 @@ echo "config files: ok"
 export KUBECONFIG="${HOME}/.kube/non-default-config"
 export CLUSTER_ADMIN_CONTEXT=$(oc config view --flatten -o template -t '{{index . "current-context"}}')
 
+pushd "$OS_ROOT" > /dev/null
+restore_nullglob=$(shopt -p nullglob)
```

Comment:
It would help if you explained why the find is broken.  Since we use it extensively elsewhere in the test scripts, it's likely that'll benefit the folks reading this.  Our general answer for bash is not RTFM - it's WTFAYUBash.  Using the minimal set of features, clearly explaining why code is doing something, and using a set of common idioms has been our rule of thumb so far.